app/src/main/java/com/tpp/theperiodpurse/ui/onboarding/QuestionOneScreen.kt

```diff
@@ -1,90 +1,201 @@
+                        .size(imageheight.dp)
+                        .align(Alignment.Center),
+                )
+                val barheight1 = (screenheight * (0.09))
 
-    Column(
-        modifier = Modifier.padding(32.dp),
-        horizontalAlignment = Alignment.CenterHorizontally,
-        verticalArrangement = Arrangement.spacedBy(8.dp)
-
-    ) {
-        Text(
-            text = stringResource(R.string.question_one),
-            fontSize = 24.sp,
-            modifier = Modifier.align(Alignment.CenterHorizontally)
-        )
-        Text(
-            text = stringResource(R.string.description_one),
-            fontSize = 15.sp,
-            modifier = Modifier.align(Alignment.CenterHorizontally)
-        )
-        Spacer(Modifier.height(16.dp))
-
-        EditNumberField(
-            label = R.string.tap_to_input,
-            value = periodCycle,
-            onValueChange = { periodCycle = it },
-
-            keyboardOptions = KeyboardOptions(
-                keyboardType = KeyboardType.Number,
-                imeAction = ImeAction.Done
-            ),
-            keyboardActions = KeyboardActions(
-                onDone = { focusManager.clearFocus() ; onSelectionChanged(periodCycle) ; entered = true })
-        )
-        Spacer(Modifier.height(24.dp))
+                Image(
+                    painter = painterResource(R.drawable.onboard_bar1),
+                    contentDescription = null,
+                    modifier = Modifier
+                        .size(barheight1.dp)
+                        .align(Alignment.BottomCenter),
+                )
+
+            }
+
+            Text(
+                text = stringResource(R.string.question_one),
+                fontSize = 28.sp,
+                modifier = Modifier
+                    .align(Alignment.CenterHorizontally)
+                    .width(250.dp),
+                fontWeight = FontWeight.Bold,
+                textAlign = TextAlign.Center
+            )
+            Spacer(Modifier.height((screenheight * (0.02)).dp))
+
+            Text(
+                text = stringResource(R.string.description_one),
+                fontSize = 18.sp,
+                modifier = Modifier
+                    .align(Alignment.CenterHorizontally)
+                    .width((screenwidth * (0.6)).dp),
+                textAlign = TextAlign.Center
+            )
+            Spacer(Modifier.height((screenheight * (0.02)).dp))
+
+            EditDaysField(
+                label = R.string.tap_to_input,
+                value = periodCycle,
+                onValueChange = { periodCycle = it },
+
+                keyboardOptions = KeyboardOptions(
+                    keyboardType = KeyboardType.Number,
+                    imeAction = ImeAction.Done
+                ),
+                keyboardActions = KeyboardActions(
+                    onDone = {
+                        focusManager.clearFocus(); onSelectionChanged(periodCycle); entered = true
```

Comment:
On this page, if the input is empty and I click done, the app still crashes. Can you please write tests for testing inputs of various kinds to make sure this doesn't happen?